.vscode/settings.template.json

```diff
@@ -20,6 +20,12 @@
         "organizeImportsIgnoreCase": false,
         "organizeImportsNumericCollation": true
     },
+    "javascript.unstable": {
```

Comment:
While at it can you add comma to gitlens options so we dont need to fix that everytime copying these and uncommenting.

 // "gitlens.advanced.blame.customArguments": [
    //     "--ignore-revs-file",
    //     ".git-blame-ignore-revs"
    // ]

Reply:
Bah, race condition, I just merged it. I can send another PR for this.